src/components/BlogCard.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="blog-card-container">
+    <div class="blog-card-content">
+      <h4 class="blog-content-author-name">{{ authorName }}</h4>
+      <div>
+        <h2 class="blog-content-title">{{ title }}</h2>
+      </div>
+      <div class="blog-content-summary-container">
+        <h3 class="blog-content-summary">{{ summary }}</h3>
+      </div>
+      <div>
+        <span class="blog-date">{{ publishedDate }}</span>
+      </div>
+    </div>
+    <div class="blog-card-img-container">
+      <img class="blog-card-img" :src="imageURL" alt="Blog" />
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    imageURL: String,
+    title: String,
+    summary: String,
+    category: String,
+    authorName: String,
+    publishedDate: String,
+  },
+}
+</script>
+
+<style scoped>
+.blog-card-container {
+  width: 25rem;
+  display: flex;
+  min-width: 20rem;
```

Comment:
I don't think it can be 20 rem or 25 rem. rem depends on the root element's font size. Please handle this issue throughout the css. 
Go through this link and understand how it works. Whether you need to rem or em https://webdesign.tutsplus.com/tutorials/comprehensive-guide-when-to-use-em-vs-rem--cms-23984